packages/plugin-react/src/index.ts

```diff
@@ -40,6 +40,11 @@ export interface Options {
    * @default true
    */
   jsxPure?: boolean
+  /**
+   * Toggles whether or not to throw an error if an XML namespaced tag name is used.
+   * @default true
+   */
+  throwIfNamespace?: boolean
```

Comment:
I think this should be prefixed aligning it to other flags 
```suggestion
  jsxThrowIfNamespace?: boolean
```
Maybe at one point in the future, it would be better to have all these flags nested under a `jsx` object (cc @aleclarson)

Reply:
> I think this should be prefixed aligning it to other flags

@patak-dev Alright!

> it would be better to have all these flags nested under a jsx object

Yeah, that's even better, but it would be a breaking change.